src/components/Opportunity/ActorWhiteboard.tsx

```diff
@@ -0,0 +1,175 @@
+import Excalidraw, { serializeAsJSON } from '@excalidraw/excalidraw';
+import { ExcalidrawAPIRefValue, ExcalidrawProps } from '@excalidraw/excalidraw/types/types';
+import { makeStyles } from '@mui/styles';
+import { isNil } from 'lodash';
+import React, { FC, useEffect, useRef, useState } from 'react';
+import { useApolloErrorHandler } from '../../hooks';
+import { useUpdateActorMutation, useUpdateEcosystemModelMutation } from '../../hooks/generated/graphql';
+import { Actor, EcosystemModel, Maybe } from '../../models/graphql-schema';
+import { NewWhiteboardActorModal } from './NewWhiteboardActorModal';
+
+interface ActorWhiteboardProps {
+  actors: Actor[];
+  ecosystemModel: Maybe<EcosystemModel>;
+}
+
+const useActorWhiteboardStyles = makeStyles(_theme => ({
+  container: {
+    height: 600,
+    excalidrawWrapper: true,
+  },
+}));
+
+const initialExcalidrawState = {
+  type: 'excalidraw',
+  version: 2,
+  source: 'https://excalidraw.com',
+  elements: [],
+  appState: {
+    gridSize: null,
+    viewBackgroundColor: '#ffffff',
+  },
+  files: {},
+};
+
+const ActorWhiteboard: FC<ActorWhiteboardProps> = ({ actors = [], ecosystemModel }) => {
```

Comment:
This is a legacy component that will be used as an example when we create a canvas template that operates with the ecosystem model.